code/lib/blocks/src/controls/options/RadioOptions.stories.tsx

```diff
@@ -79,7 +79,7 @@ export const Object: Story = {
   name: 'DEPRECATED: Object',
   args: {
     value: objectOptions.B,
-    argType: { options: objectOptions },
+    argType: { options: objectOptions as any },
```

Comment:
This is technically a breaking change. We have to ensure that passing options the deprecated way is still supported and shouldn't trigger type issues for our users. We can definitely make the above type changes in Storybook 9, but not now. Can you adjust the PR to not throw away the custom types in `code/lib/blocks/src/components/ArgsTable/types.ts`, but leave a note instead to remove them in Storybook 9?

Reply:
@valentinpalkovic - i _think_ you made this change to that file did you not? 🤔 [9f440259ec06dc5c1faf336bb67b431ca10a041e](https://github.com/storybookjs/storybook/pull/28718/commits/9f440259ec06dc5c1faf336bb67b431ca10a041e)

Reply:
Hehe. I knew the changes were kind of familiar to me xD. I'll take another look :)

Reply:
> Hehe. I knew the changes were kind of familiar to me xD. I'll take another look then.

Heh no worries, I have weeks/months like that. Looked at so many code/projects I can't remember what I've done 😆